src/Screens/Main/Chat/index.tsx

```diff
@@ -168,6 +172,10 @@ const Chat = ({ navigation }: Props) => {
     }
   }, [isMessageSendFailed]);
 
+  React.useEffect(() => {
+    getDraftMessage();
+  }, [buddyId]);
```

Comment:
We probably want to run this only once?

This only re-runs when buddyId !== buddyId before so this probably is fine.

Reply:
Yeah, this probably runs only when user changes chat-view, also then the buddyId changes. I can leave the dependency array-empty